src/tcgwars/logic/impl/gen8/RebelClash.groovy

```diff
@@ -3359,9 +3359,10 @@ public enum RebelClash implements LogicCardInfo {
         move "Chirp", {
           text "Search your deck for up to 2 Pokemon with a [F] Resistance, reveal them, and put them into your hand. Then, shuffle your deck."
           energyCost C
-          attackRequirement {}
+          attackRequirement {
+            assert my.deck : "There are no more cards in your deck."}
           onAttack {
-            // TODO
+            deck.search (max: 2 ,{it.cardTypes.is(POKEMON) &&  it.asPokemonCard().resistance.contains(F)}).showToOpponent("Chosen Pokémon cards.").moveTo(my.hand)
```

Comment:
What do you think of not using showToOpponent? The search action will still appear in the game log.  I've personally found the showToOpponent thing annoying but I'd like to hear your opinion

Reply:
Personally I appreciate showToOpponent as it lets you read the cards found before play resumes. It lessens the burden on players to know exactly what cards do just from their name.